src/h.js

```diff
@@ -1,3 +1,25 @@
+var nodePool = []
+var nodePoolHead = -1
```

Comment:
should be fine, these are nodes that should no longer be used as we had been just throwing them away to GC